scene/2d/audio_stream_player_2d.cpp

```diff
@@ -313,11 +312,15 @@ void AudioStreamPlayer2D::stop() {
 
 bool AudioStreamPlayer2D::is_playing() const {
 
+#ifdef TOOLS_ENABLED
```

Comment:
These should not be under TOOLS_ENABLED, as this would affect running the game from inside the editor also. This should be a check for Engine::get_singleton()->is_editor_hint(). The actual runtime check could still be inside an #ifdef block.